frontend/src/cards/ui/MultiMapDialog.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import Alert from "@material-ui/lab/Alert";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import { Grid } from "@material-ui/core";
+import { ChoroplethMap } from "../../charts/ChoroplethMap";
+import { Fips } from "../../data/utils/Fips";
+import { Legend } from "../../charts/Legend";
+import { MetricConfig } from "../../data/config/MetricConfig";
+import { Row, FieldRange } from "../../data/utils/DatasetTypes";
+import styles from "./MultiMapDialog.module.scss";
+
+export interface MultiMapDialogProps {
+  metricConfig: MetricConfig;
+  breakdown: string;
+  fips: Fips;
+  data: Row[];
+  fieldRange: FieldRange | undefined;
+  open: boolean;
+  handleClose: () => void;
+  breakdownValues: string[];
+}
+
+export function MultiMapDialog(props: MultiMapDialogProps) {
```

Comment:
Add comment for what this is (I think this is a good habit to start getting into for components unless it's really obvious).

Same for the legend and two-level dropdown

Doesn't need a comment for every prop, but would be good to explain how the major behavior changing ones work. Eg breakdown & breakdownValues